### PR TITLE
prepare release 1.6.2

### PR DESCRIPTION
**Pull request by luser**
- update changelog for 1.6.2
- Release version 1.6.2

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 1.6.2
+
+- Load `fs-debug.js` capture script when `debug = true` option is used to match updated debug logging practices.
```
Comment (patrick-fs) [inline CHANGELOG.md:5]: Is there a KB article or some external doc that we can link to?

Comment (luser) [reply CHANGELOG.md:5]: Yeah, I linked it in the PR and such when I implemented this. I'll add it here for the sake of completeness.

**Review by patrick-fs — APPROVED** [review]
LGTM

**Comment by luser** [comment]
https://www.npmjs.com/package/@fullstory/browser/v/1.6.2

**Pull request merged**